source/configuration_manual/sieve/configuration.rst

```diff
@@ -312,6 +312,33 @@ Configurable Limits
    prohibited. For older versions, however, this means that the number
    of redirects is *unlimited*, so be careful.
 
+:pigeonhole:ref:`sieve_max_cpu_time` = 30s (v0.5.15+)
+   The maximum amount of CPU time that a Sieve script is allowed to use while
+   executing. If the execution exceeds this resource limit, the script ends with
+   an error, causing the implicit "keep" action to be executed. This limit is
+   not only enforced for a single script execution, but also cumulatively for
+   the last executions within a configurable timeout
+   (see :pigeonhole:ref:`sieve_resource_usage_timeout <sieve_resource_usage_timeout>`).
+
+:pigeonhole:ref:`sieve_resource_usage_timeout` = 1h (v0.5.15+)
+   To prevent abuse, the Sieve interpreter can record resource usage of a Sieve
+   script execution in the compiled binary if it is significant. Currently, this
```

Comment:
Isn't this too much copy&pasting the same text to two locations here? How is it intended to be handled nowadays? @slusarz

Reply:
@slusarz ping

Reply:
All configuration items - or at least a link to all available configuration items  - should go in source/settings/pigeonhole.rst.  There remains a bunch of this kind of duplication in pigeonhole code, since I concentrated on moving all config items to a single location (using the new macros) rather than trying to cleanup each individual page.

I think it's fine to keep a collection of specific configuration settings on a subpage, if it works better there.  See, e.g., imapc settings (https://doc.dovecot.org/configuration_manual/mail_location/imapc/) for core.  But if we do that, it is important to create a link on the main source/settings/pigeonhole.rst page to the setting - the main configuration page is intended to display ALL possible settings in a single location.

Agreed that the description of a setting should only be maintained in a single location.

Reply:
What should I do?

Reply:
ping @slusarz

Reply:
The issue is that sieve_max_cpu_time (and sieve_resource_usage_timeout) is now being documented in two different places. All text should be moved under the configuration option and the other page should link to the setting.

Reply:
better?

Reply:
We could of course just merge this current pull request and try to improve things later..

Reply:
This has taken forever, so lets just merge this.